rhobs/alerting/data_plane/prometheus.latency_release_creation_alerts.yaml

```diff
@@ -0,0 +1,33 @@
+apiVersion: monitoring.coreos.com/v1
+kind: PrometheusRule
+metadata:
+  name: rhtap-latency-release-creation-alerting-rules
+  labels:
+    tenant: rhtap
+spec:
+  groups:
+  - name: latency-release-creation
+    interval: 1m
+    rules:
+    - alert: LatencyReleaseCreation
+      expr: |
+       (
+        (
+         increase(release_latency_seconds_bucket{le="+Inf"}[5m])
+         - ignoring(le)
+         increase(release_latency_seconds_bucket{le="10"}[5m])
+        ) / ignoring(le)
+        increase(release_latency_seconds_bucket{le="+Inf"}[5m])
+       ) > 0.10
+      for: 1m
+      labels:
+        severity: warning
+        slo: true
+      annotations:
+        summary: >-
+          Snapshot marked as passed to Release created time exceeded
+        description: >
```

Comment:
I see more than one result per cluster. If they are not to be aggregated then I think the description should include the label that sets them apart.

Reply:
done, thank you.

Reply:
I don't see it.

Reply:
It was uploading as I left you the message. You are too efficient!! 
https://github.com/redhat-appstudio/o11y/pull/112/files

Reply:
Consider this query:
```
release_latency_seconds_bucket{source_cluster="stone-prd-rh01", le="10"}
```
If I run it in AppSRE Grafana, I get two results. It means that there's at least one label which set those two apart.
It also means that if your current alert will fire, you won't be able to tell which of those queries had an issue.

Now the question is whether the two needs to be considered together somehow, and then they need to be aggregated (e.g. with `sum`), or each of them should stand on its own, and then we need to add some more labels to the alert description or summary so we'll be able to tell which was is at fault.

Reply:
currently on our own grafana dashboard we have just one output to our graphs so we are aggregating. This is our query for the time series graph histogram_quantile(0.90, sum(rate(release_latency_seconds_bucket[$__rate_interval])) by (le)).

Reply:
The alert is ultimately going to run on the data series visible in AppSRE Grafana so need to make it applicable there. The query above is also not what we have on the alert rule in this PR.

Reply:
@jencull what was the outcome here?

The `release_latency_seconds_bucket` metric is [collected](https://grafana.app-sre.devshift.net/explore?orgId=1&left=%7B%22datasource%22:%22P22466E8E7855F1E0%22,%22queries%22:%5B%7B%22refId%22:%22A%22,%22datasource%22:%7B%22type%22:%22prometheus%22,%22uid%22:%22P22466E8E7855F1E0%22%7D,%22editorMode%22:%22code%22,%22expr%22:%22%20%20%20%20%20%20%20%28%5Cn%20%20%20%20%20%20%20%20%28%5Cn%20%20%20%20%20%20%20%20%20increase%28release_latency_seconds_bucket%7Ble%3D%5C%22%2BInf%5C%22%7D%5B5m%5D%29%5Cn%20%20%20%20%20%20%20%20%20-%20ignoring%28le%29%5Cn%20%20%20%20%20%20%20%20%20increase%28release_latency_seconds_bucket%7Ble%3D%5C%2210%5C%22%7D%5B5m%5D%29%5Cn%20%20%20%20%20%20%20%20%29%20%2F%20ignoring%28le%29%5Cn%20%20%20%20%20%20%20%20increase%28release_latency_seconds_bucket%7Ble%3D%5C%22%2BInf%5C%22%7D%5B5m%5D%29%5Cn%20%20%20%20%20%20%20%29%22,%22legendFormat%22:%22__auto%22,%22range%22:true,%22instant%22:true%7D%5D,%22range%22:%7B%22from%22:%22now-1h%22,%22to%22:%22now%22%7D%7D) for both the integration and release services. Is this expected?

Reply:
@yftacherzog I brought this to the team yesterday and the choice was cluster names. I have amended the PR now to reflect this. 

I provided the time series graph SLO query yesterday, apologies. This is our violations panel query: 

sum(rate(release_latency_seconds_bucket{le="10"}[$__rate_interval])) by (job) / sum(rate(release_latency_seconds_count[$__rate_interval])) by (job)*100

however, based on @amisstea comment it now looks like that other metric might be from another team, release service not integration service? if i am understanding that correctly.

All of my team are on PTO today for a public holiday so I will not be able to get clarification on this. It could be that rewriting our metric and changing the name will resolve our issues. I will mark this as a WIP and follow up on Monday, you are agreeable.

Reply:
In that case, I suggest changing the query so it only picks your own metrics.

Reply:
I can't see the change.

Reply:
the change is the prefix integration_svc_ in the expression https://github.com/redhat-appstudio/o11y/pull/112/files We rewrote the metrics to differentiate between us and release service

Reply:
oh. I see now.